src/amplitude.js

```diff
@@ -290,6 +290,21 @@ define(function (require) {
     }
   };
 
+/**
+   *  bufferSize of Amplitude 
+   *  @method setBufferSize
+   *  @for p5.Amplitude
+   *  @param {Number} set bufferSize to any value multiple of 2
+   */
+
+  p5.Amplitude.prototype.setBufferSize = function(s) {
+    if (s%2==0) {
+      this._workletNode.port.postMessage({ name: 'bufferSize', bufferSize: s });
+    } else {
+      console.log('Error: bufferSize must be multiple of 2');
```

Comment:
are we checking for a multiple of 2, or power of 2? Are there different min/max limits for different browsers?